Sources/ExyteChat/ChatView/Recording/RecordWaveform.swift

```diff
@@ -12,6 +12,9 @@ struct RecordWaveformWithButtons: View {
     @Environment(\.chatTheme) private var theme
 
     @StateObject var recordPlayer = RecordingPlayer()
+    //160 is screen left-padding/right-padding and playButton's width.
+    //To ensure that the view does not exceed the screen, need to subtract
+    static let viewPadding:CGFloat = 160
```

Comment:
how can you guarantee that this will always be 160? you should pass this value using frame getters from wherever it is set in